readme.md

```diff
@@ -45,6 +45,7 @@ Returns the result `Object`.
 - `title` - Window title
 - `id` - Window ID
 - `app` - App owning the window
+- `appPath` - on Windows only returns the path to the app executable
```

Comment:
I'd write it like
```
- `appPath` - Path to the app executable (Windows only)
````

Reply:
Agreed. Not sure what I was thinking when I wrote it like that.